MCP_Server/server.py

```
@@ -406,6 +487,163 @@ def set_tempo(ctx: Context, tempo: float) -> str:
+    """
+    try:
+        ableton = get_ableton_connection()
+        result = ableton.send_command("set_master_volume", {"volume": volume})
+        return f"Set master volume to {volume}"
+    except Exception as e:
+        logger.error(f"Error setting master volume: {str(e)}")
+        return f"Error setting master volume: {str(e)}"
+
+
+@mcp.tool()
+def create_audio_effect_rack(ctx: Context, track_index: int, device_index: int = -1) -> str:
```

Ruff (0.14.6)

Unused function argument: `ctx`

(ARG001)